examples/utils.py

```diff
@@ -95,6 +107,16 @@ def load_dataset(args):
     from deepchem.molnet import load_lipo
     tasks, all_dataset, transformers = load_lipo(
         featurizer=featurizer, splitter=splitter, reload=False)
+  elif args['dataset'] == 'PDBbind':
+    from deepchem.molnet import load_pdbbind
+    tasks, all_dataset, transformers = load_pdbbind(
+        featurizer=featurizer,
+        save_dir='.',
+        data_dir='.',
+        splitter='random',
```

Comment:
Any deterministic split for PDBbind?

Reply:
Not yet, but there is a plan to work on splits based on interaction fingerprints and ligand fingerprints!